src/distances/sinus.jl

```diff
@@ -0,0 +1,13 @@
+struct Sinus{T} <: Distances.PreMetric
```

Comment:
By using
```suggestion
struct Sinus{T} <: Distances.SemiMetric
```
Distances exploits the symmetry when calling `pairwise`.

Reply:
Thanks I will also use it for `DotProduct` then